src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -71,55 +75,50 @@
     private final UseRestrictionValidatorAPI useRestrictionValidatorAPI;
     private final DACUserAPI dacUserAPI;
     private final ElectionAPI electionAPI;
+    private final GCSStore store;
 
-    public DataAccessRequestResource(DACUserAPI dacUserAPI, ElectionAPI electionAPI) {
+    public DataAccessRequestResource(DACUserAPI dacUserAPI, ElectionAPI electionAPI, GCSStore store) {
         this.dataAccessRequestAPI = AbstractDataAccessRequestAPI.getInstance();
         this.consentAPI = AbstractConsentAPI.getInstance();
         this.matchProcessAPI = AbstractMatchProcessAPI.getInstance();
         this.emailApi = AbstractEmailNotifierAPI.getInstance();
         this.useRestrictionValidatorAPI = AbstractUseRestrictionValidatorAPI.getInstance();
         this.dacUserAPI = dacUserAPI;
         this.electionAPI = electionAPI;
+        this.store = store;
     }
 
     @POST
     @Consumes("application/json")
     @Produces("application/json")
     @RolesAllowed("RESEARCHER")
     public Response createdDataAccessRequest(@Context UriInfo info, Document dar) {
-        URI uri;
-        List<Document> result;
         UseRestriction useRestriction;
         try {
             Boolean needsManualReview = DarUtil.requiresManualReview(dar);
-            if (!needsManualReview) {
-                // generates research purpose, if needed, and store it on Document rus
-                useRestriction = dataAccessRequestAPI.createStructuredResearchPurpose(dar);
-                dar.append(DarConstants.RESTRICTION, Document.parse(useRestriction.toString()));
-                useRestrictionValidatorAPI.validateUseRestriction(useRestriction.toString());
-                dar.append(DarConstants.VALID_RESTRICTION, true);
+            try {
+                if (!needsManualReview) {
+                    useRestriction = dataAccessRequestAPI.createStructuredResearchPurpose(dar);
+                    dar.append(DarConstants.RESTRICTION, Document.parse(useRestriction.toString()));
+                    useRestrictionValidatorAPI.validateUseRestriction(useRestriction.toString());
+                    dar.append(DarConstants.VALID_RESTRICTION, true);
+                }
+            } catch (Exception e) {
+                logger.log(Level.SEVERE, "while creating useRestriction " + dar.toJson(), e);
```

Comment:
nit: Error message could use some help
```suggestion
                logger.log(Level.SEVERE, "Error creating use restriction for data access request " + dar.toJson(), e);
```